docs/farming-&-staking/farming/additional-guides/grafana-dashboard.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Grafana Dashboard
+sidebar_position: 2
+description: How to configure the Grafana dashboard for a Subspace Network node
+keywords:
+    - node
+    - farmer
+    - monitoring
+    - grafana
+    - prometheus
+---
+
+# Grafana Dashboard
+
+## Prerequisites
+
+First install [Grafana](https://grafana.com/) and [Prometheus](https://prometheus.io/) on your machine.
+
+:::note
+You can also use [Grafana Cloud](https://grafana.com/products/cloud/) instead of installing it locally.
+:::
+
+## Enable metrics
+
+To enable metrics, you'll have to modify the CLI arguments of your node and farmer by adding the Prometheus option:
+
+for a node:
+`--prometheus-listen-on 127.0.0.1:9080`
+
+for a farmer:
+`--metrics-endpoints 127.0.0.1:9081`
+
+To apply the changes, it's necessary to restart your node and farmer. You should then have the capability to reach the metrics at `http://localhost:9080` and `http://localhost:9081`.
+
+## Configure Prometheus targets
+
+Locate the `prometheus.yml` file and add the following job entry under the `scrape_configs` section:
+```
+- job_name: "subspace"
+  static_configs:
+    - targets: ["localhost:9080"]
+      labels:
+        group: 'node'
+    - targets: ["localhost:9081"]
+      labels:
+        group: 'farmer'
+```
+On Linux machines, you can typically find the `prometheus.yml` file at the following path: `/etc/prometheus/prometheus.yml`
+
+Restart Prometheus with the new configuration.
+
+:::note
+Prometheus can reload its configuration without restarting by using the SIGHUP signal. If you're running on Linux this can be performed by using `kill -s SIGHUP <PID>`, replacing <PID> with your Prometheus process ID.
```

Comment:
```suggestion
Prometheus can reload its configuration without restarting by using the SIGHUP signal. If you're running on Linux this can be performed by using `kill -s SIGHUP <PID>`, replacing `<PID>` with your Prometheus process ID.
```

This <PID> is breaking a preview, let's please fix it and have another glance how it looks on a preview deployment.
I did build it locally and it seemed fine 👍

Reply:
fixed